packages/web/src/js/Collapse.ts

```diff
@@ -59,100 +62,133 @@ class Collapse extends BaseComponent {
     return `${this.NAME}`;
   }
 
-  onResize() {
-    this.state.width = window.innerWidth;
-    this.updateTriggerElementHandler();
-    this.updateCollapsibleElementHandler();
+  siblingsControlHandler(trigger: HTMLElement) {
```

Comment:
I suggest renaming this function. It is not an event handler passed as a callback, so starting with a verb will better describe what the function does. Eg. `hideParentSiblings` or something similar?

Reply:
fixed